nhrpd/nhrp_multicast.c

```diff
@@ -130,21 +127,18 @@ static void netlink_mcast_log_handler(struct nlmsghdr *msg, struct zbuf *zb)
 		}
 	}
 
-	if (!out_ndx || !zbuf_used(&pktpl))
+	if (!out_ndx || !ctx.pkt)
 		return;
 
-	ifp = if_lookup_by_index(htonl(*out_ndx), VRF_DEFAULT);
-	if (!ifp)
+	ctx.ifp = if_lookup_by_index(htonl(*out_ndx), VRF_DEFAULT);
+	if (!ctx.ifp)
 		return;
 
-	debugf(NHRP_DEBUG_COMMON, "Outgoing interface = %s\n", ifp->name);
-
-	ctx = (struct mcast_ctx){
-		.ifp = ifp, .pkt = &pktpl,
-	};
+	debugf(NHRP_DEBUG_COMMON, "Received multicast packet on %s len %zu\n",
+		   ctx.ifp->name, zbuf_used(ctx.pkt));
 
```

Comment:
- \n is not necessary.
- also the comment may be more explicit. I did not realize at first, but we need to understand that this is a packet going out that is intercepted ( and not incoming ospf messages).
what about mesage like:
"Intercepted multicast packet leaving %s len %zu"

Reply:
missing this comment ?

Reply:
thanks